.gitmodules

```diff
@@ -22,3 +22,6 @@
 	path = libs/core2
 	url = https://github.com/theseus-os/core2.git
 	shallow = true
+[submodule "ports/rust"]
+	path = ports/rust
+	url = https://github.com/theseus-os/rust.git
```

Comment:
let's not make this a submodule, but instead clone it (if it doesn't already exist) as part of a separate makefile target (on which the `std` target would depend).

Reply:
Yea, your review is of an older commit.